pkg/registry/apis/provisioning/repository/github.go

```diff
@@ -223,6 +223,16 @@ func (r *githubRepository) Create(ctx context.Context, path, ref string, data []
 	owner := r.config.Spec.GitHub.Owner
 	repo := r.config.Spec.GitHub.Repository
 
+	// Create .keep file if it is a directory
+	if strings.HasSuffix(path, "/") {
+		if data != nil {
+			return apierrors.NewBadRequest("data cannot be provided for a directory")
+		}
+
+		path = strings.TrimSuffix(path, "/") + "/.keep"
```

Comment:
are we using .keep anywhere, or is that new?

I wonder about using `README.md` for github -- seems like a good place to write a title/description if we need to 🤔

Reply:
@ryantxu 
no, we are not using it. `.keep` file is the typical way people force git to track the folder. 

The advantage of using `.keep` apart from following the convention is that you are sure that it does not contain anything. A `README.md` normally contains some information written by the somebody and we couldn't remove it once created (easily and securely). 


I think that the `README.md` may be a good nice-to-have idea for the exporting as we could create a nice markdown with some information.